clouditor-engine-core/src/main/java/io/clouditor/data_access_layer/HibernateUtils.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2016-2019 Fraunhofer AISEC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *            $$\                           $$\ $$\   $$\
+ *            $$ |                          $$ |\__|  $$ |
+ *   $$$$$$$\ $$ | $$$$$$\  $$\   $$\  $$$$$$$ |$$\ $$$$$$\    $$$$$$\   $$$$$$\
+ *  $$  _____|$$ |$$  __$$\ $$ |  $$ |$$  __$$ |$$ |\_$$  _|  $$  __$$\ $$  __$$\
+ *  $$ /      $$ |$$ /  $$ |$$ |  $$ |$$ /  $$ |$$ |  $$ |    $$ /  $$ |$$ |  \__|
+ *  $$ |      $$ |$$ |  $$ |$$ |  $$ |$$ |  $$ |$$ |  $$ |$$\ $$ |  $$ |$$ |
+ *  \$$$$$$\  $$ |\$$$$$   |\$$$$$   |\$$$$$$  |$$ |  \$$$   |\$$$$$   |$$ |
+ *   \_______|\__| \______/  \______/  \_______|\__|   \____/  \______/ \__|
+ *
+ * This file is part of Clouditor Community Edition.
+ */
+
+package io.clouditor.data_access_layer;
+
+import io.clouditor.assurance.*;
+import io.clouditor.assurance.ccl.AssetType;
+import io.clouditor.assurance.ccl.Condition;
+import io.clouditor.assurance.ccl.FilteredAssetType;
+import io.clouditor.auth.User;
+import io.clouditor.discovery.Asset;
+import io.clouditor.discovery.DiscoveryResult;
+import io.clouditor.discovery.Scan;
+import java.util.Objects;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+/**
+ * An Utility class to initialize and configure Hibernate.
+ *
+ * @author Andreas Hager, andreas.hager@aisec.fraunhofer.de
+ */
+public class HibernateUtils {
+
+  private HibernateUtils() {}
+
+  private static final Configuration CONFIGURATION =
+      new Configuration()
+          .configure()
+          // Add all Entities.
+          .addAnnotatedClass(Domain.class)
```

Comment:
Is it possible to add a package rather than individual classes?

Reply:
No, not directly. You can write a method, that walks trugh the package and looks for the @Entity annontation, e.g. with https://github.com/ronmamo/reflections.